commit-message.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines to create great commit messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
```

Comment:
Yeah I think GitHub warns at 50 characters for the subject but accepts more than that.
In fact I think it accepts anything when you push.

The guidelines for the Git project are 80 characters for everything I think (code, comments, commit messages).